iis-wf/iis-wf-primary/src/main/resources/eu/dnetlib/iis/wf/primary/main/oozie_app/workflow.xml

```diff
@@ -819,10 +819,30 @@
                 </property>
             </configuration>
         </sub-workflow>
-        <ok to="end" />
+        <ok to="build-report" />
         <error to="fail" />
     </action>
 
+    <action name="build-report">
```

Comment:
Since we have `preprocessing` alongside `primary` workflow we should generate reports for both. Could you define it there as well? If you think this task deserves creating additional ticket, please do create one.